ngi_reports/common/ign_sample_report.py

```diff
@@ -216,6 +217,34 @@ def parse_snpeff(self):
 
 
 
+    def parse_picard_metrics(self):
+        """ Parse the picard metrics file to get the duplication rates
+        """
+        for sample_id in self.samples.iterkeys():
+
+            # Build the expected filenames
+            picard_metrics_fn = os.path.realpath(os.path.join(self.working_dir,
+                '05_processed_alignments', '{}.metrics'.format(sample_id)))
+            try:
+                synonymous_SNPs = 0
```

Comment:
Argh, yes. Must read code..

I've started modifying this script elsewhere now, I've removed these lines for the next commit though. Could you merge this anyway so that it doesn't get confusing?

Sorry...